### PR TITLE
Feature9 add readme

### LINKED ISSUE #9: Add Readme

### PR DESCRIPTION
**Pull request by JoyceXinyueWang**
In this pull request, I updated ReadMe with renovated anonlink-client as a client-face API to interact with anonlink system.

The documentation for CLI tools is comprehensive in clkhash readthedocs. So I pointed the documentation there. In the future, We might need to create a readthedocs for anonlink-client and migrate the docs from clkhash to anonlink-client.

Close #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
+
+The way to interact with anonlink system is via Command Line Tool `anonlink`. You can hash data containing PII (Personal
+ Identifying Information) locally using `anonlink hash`, generate candidate blocks locally to scale up record linkage 
+ using `anonlink block`, create a record linkage job in entity service with `anonlink create-project` etc.
+
+### Installation
+Currently manual install:
+
+```python3
+pip install git+https://https://github.com/data61/clkhash
```
Comment (hardbyte) [inline README.md]: We can keep installing directly from github but use the current repository:

```suggestion
pip install git+https://https://github.com/data61/anonlink-client
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
-Two methods have been implemented that map to a list of records.
+### Documentation
+https://clkhash.readthedocs.io/en/stable/cli.html
+Note that the documentation are for clkhash now, we will add a readthedocs page for anonlink-client very soon.
+
+### CLI Tool
+After installation, you should have a `anonlink` program in your path. For
+example, to hash PII data  `alice.csv` locally with schema `schema.json` and secret `horse`, run:
+```bash
+$ anonlink hash 'alice.csv' 'horse' 'schema.json' 'clk.json'
```
Comment (hardbyte) [inline README.md]: Perhaps a good time to rename `hash` to `encode`? @wilko77

Comment (hardbyte) [reply README.md]: I wouldn't use the filename `clk.json` either, perhaps `encoded-entities.json`

Comment (JoyceXinyueWang) [reply README.md]: An issue is created for the renaming proposal #15

**Pull request merged**